ansible/roles/cli/tasks/download_openwhisk_cli.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+# Download the cli to Nginx CLI directory
+
+- set_fact:
+    cli_tag="{{ openwhisk_cli.tag }}"
+    arc_64="amd64"
+    arc_32="386"
+
+- name: "ensure Nginx cli directory for ({{ item }}) amd64 exists"
```

Comment:
Seems like you could just the same functions to handle amd64 and 386 instead of duplicating the script for each.